demo/src/app/components/nav/demos/config/nav-config.html

```diff
@@ -0,0 +1,16 @@
+<ul ngbNav #nav="ngbNav">
```

Comment:
this requires `class="nav-tabs"` (otheriwse the active one is not rendered), or type-base implementation on `ngbNav` as in tabset

Reply:
Fixed, thanks